pkg/antctl/raw/featuregates/command_test.go

```diff
@@ -270,19 +270,19 @@ func TestGetFeatureGates(t *testing.T) {
 		{
 			name:           "get featuregates out of Pod",
 			runE:           controllerRemoteRunE,
-			expectedOutput: "Antrea Agent Feature Gates\nFEATUREGATE              STATUS         VERSION   \nEndpointSlice            Disabled       ALPHA     \nAntreaIPAM               Disabled       ALPHA     \nFlowExporter             Disabled       ALPHA     \nNetworkPolicyStats       Enabled        BETA      \nAntreaPolicy             Enabled        BETA      \nMulticast                Disabled       ALPHA     \nTraceflow                Enabled        BETA      \nServiceExternalIP        Disabled       ALPHA     \nAntreaProxy              Enabled        BETA      \nEgress                   Enabled        BETA      \nNodePortLocal            Enabled        BETA      \nMulticluster             Enabled        ALPHA     \nAntrea Controller Feature Gates\nFEATUREGATE              STATUS         VERSION   \nAntreaPolicy             Enabled        BETA      \nNodeIPAM                 Disabled       ALPHA     \nTraceflow                Enabled        BETA      \nServiceExternalIP        Disabled       ALPHA     \nEgress                   Enabled        BETA      \nNetworkPolicyStats       Enabled        BETA      \n",
+			expectedOutput: "Antrea Agent Feature Gates\nFEATUREGATE                   STATUS         VERSION   \nEndpointSlice                 Disabled       ALPHA     \nAntreaIPAM                    Disabled       ALPHA     \nFlowExporter                  Disabled       ALPHA     \nNetworkPolicyStats            Enabled        BETA      \nAntreaPolicy                  Enabled        BETA      \nMulticast                     Disabled       ALPHA     \nTraceflow                     Enabled        BETA      \nServiceExternalIP             Disabled       ALPHA     \nAntreaProxy                   Enabled        BETA      \nEgress                        Enabled        BETA      \nNodePortLocal                 Enabled        BETA      \nMulticluster                  Enabled        ALPHA     \nAntrea Controller Feature Gates\nFEATUREGATE                   STATUS         VERSION   \nAntreaPolicy                  Enabled        BETA      \nNodeIPAM                      Disabled       ALPHA     \nTraceflow                     Enabled        BETA      \nServiceExternalIP             Disabled       ALPHA     \nEgress                        Enabled        BETA      \nNetworkPolicyStats            Enabled        BETA      \n",
```

Comment:
Use the expected output like following and it is more readable. So do others.
```go
expectedOutput:`Antrea Agent Feature Gates
FEATUREGATE                        STATUS         VERSION
AntreaIPAM                         Disabled       ALPHA
AntreaPolicy                       Enabled        BETA
AntreaProxy                        Enabled        BETA
Egress                             Enabled        BETA
EndpointSlice                      Disabled       ALPHA
FlowExporter                       Disabled       ALPHA
Multicast                          Disabled       ALPHA
Multicluster                       Enabled        ALPHA
NetworkPolicyStats                 Enabled        BETA
NodePortLocal                      Enabled        BETA
ServiceExternalIP                  Disabled       ALPHA
Traceflow                          Enabled        BETA
Antrea Controller Feature Gates
FEATUREGATE                        STATUS         VERSION
AntreaPolicy                       Enabled        BETA
Egress                             Enabled        BETA
NetworkPolicyStats                 Enabled        BETA
NodeIPAM                           Disabled       ALPHA
ServiceExternalIP                  Disabled       ALPHA
Traceflow                          Enabled        BETA
`
```

Reply:
fix: https://github.com/antrea-io/antrea/pull/5314/commits/e25c0ff122ea656e49641d0e3da70945475b0e69